model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -719,6 +716,18 @@ message TimestampTransform {
   }
 }
 
+// A reference to a side input id.
+message SideInputId {
+  // The id of the PTransform that references this side input.
+  string transform_id = 1;
+
+  // The local name of this side input from the PTransform that references it.
+  string local_name = 2;
+
+  // The id of the PCollection that backs this side input.
```

Comment:
I don't think you actually need this field - you can just retrieve it from the `ExecutableStage` components from https://github.com/apache/beam/pull/4948, and get the global name of the PCollection from the input PTransform

Reply:
Ah, good point.